### PR TITLE
Raidboss: Make Japanese translations more natural (STORMBLOOD)

### PR DESCRIPTION
**Pull request by nekomimi-mode**
Fixing Japanese translations of Raidboss timeline to make more natural. At this moment, most of "stormblood" contents only.

### CODE REVIEW

File: ui/raidboss/data/04-sb/raid/o3n.js [hunk 1]
```diff
@@ -331,7 +331,7 @@ export default {
         text: {
           en: 'Mindjack: Left',
           de: 'Geistlenkung: Links',
-          ja: 'マインドジャック: 左折',
+          ja: 'マインドジャック: 左方向',
```
Comment (MaikoTan) [inline ui/raidboss/data/04-sb/raid/o3n.js:334]: "左折" and "右折" is the original word in-game, do you think it is better to translate it?

File: ui/raidboss/data/04-sb/raid/o8s.js [hunk 2]
```diff
@@ -188,7 +188,7 @@ export default {
           en: 'Future: Stack and Through',
           de: 'Zukunft: Sammeln und Durchlaufen',
           fr: 'Futur : Stack et traversez',
-          ja: '未来: シェア後ボス通り抜ける',
+          ja: '未来: シェア後ボスを通り抜ける',
```
Comment (MaikoTan) [inline ui/raidboss/data/04-sb/raid/o8s.js:191]:
```suggestion
          ja: '未来: 頭割り後ボスを通り抜ける',
```
I think to describe "damage share" gimmick, we don't use "シェア" anymore, all use "頭割り" instead.

**Review by MaikoTan — APPROVED** [review]

File: ui/raidboss/data/04-sb/raid/o3n.js (hunk 1 above)
Comment (nekomimi-mode) [reply ui/raidboss/data/04-sb/raid/o3n.js]: I've reverted them.
But I think both are fine. Furthermore, only "左" or "右" is enough for this gimmick.

Comment (MaikoTan) [reply ui/raidboss/data/04-sb/raid/o3n.js]: Yep, I agree that.
For example they wrote only "左" and "右" in cn things.

I am just thinking to use the same word in-game would be more accurate.

**Pull request merged**